spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/domain/BlogPost.java

```diff
@@ -0,0 +1,92 @@
+package io.crate.spring.jdbc.samples.domain;
+
+import java.util.Date;
+
+public class BlogPost {
```

Comment:
Could we maybe make the domain model, immutable, all fields final with only getters ?

Reply:
Why?

Reply:
It is best practice. Immutable classes are in general much simpler to design, implement and use correctly.